### DISCUSSION TITLE
Linux64 app in Docker container gives Runtime error 217

### DISCUSSION THREAD

**Discussion by bert0094** [skia4delphi/skia4delphi#312] [Q&A] [answered]
RAD Studio 11.3
Compiling a project to Linux64 using PAServer on WSL with Ubuntu 20.04. I have checked "Enable Skia" for the project.
Using Skia 5.0.0.
The project runs on the WSL box.
It will not run in a Docker container with Ubuntu 20.04. I have copied libsk4d.so into the project directory in the container. I get "Runtime Error 217." 
I have seen this discussion https://github.com/skia4delphi/skia4delphi/discussions/171 but that situation appears to be different in that the apps would not run in PAServer/scratch-dir. My app does run in PAServer/scratch-dir but not in the container.
Are there prerequisites that I need to have installed in the container?

**Comment by bert0094** [comment, marked as answer]
I found I was able to run the app in a container after including the following in the dockerfile:
RUN \
  apt-get -y install util-linux &&\
  apt-get -y install libglvnd0 &&\
  apt-get -y install libfontconfig1 &&\
  apt-get -y install libx11-6 &&\
  apt-get -y install libgl1

**Reply by viniciusfbb** [reply]
@bert0094 Thanks for sharing!
